packages/cli-repl/test/e2e.spec.ts

```diff
@@ -10,6 +10,7 @@ import rimraf from 'rimraf';
 import path from 'path';
 import { readReplLogfile } from './repl-helpers';
 import { bson } from '@mongosh/service-provider-core';
+import { SIGKILL } from 'constants';
```

Comment:
`constants` is deprecated: https://nodejs.org/api/deprecations.html#DEP0008

`os.constants` is probably the better choice here

Reply:
Thanks 😅 👍